### PR TITLE
Fixes #239 - adds touch icons.

### LINKED ISSUE #239: Add apple-icons
Need to add apple-icons to root `/public` directory. “Touch icons” are the favicons of mobile devices and tablets. See [background reading](http://mathiasbynens.be/notes/touch-icons).

### PR DESCRIPTION
**Pull request by anselmbradford**
- Fixes #239 - adds touch icons.
- Edits graphics readme to remove software versions, since they can’t
be reliably checked.
- Edits CUSTOMIZE.md text to include graphics section.
- Underscores template file names to be consistent with other file structure naming conventions.

### CODE REVIEW

**Comment by anselmbradford** [comment]
@monfresh Ready for review n merge! Thx!

**Comment by anselmbradford** [comment]
Looks like:
screen shot 2015-02-16 at 7 01 34 pm

**Comment by monfresh** [comment]
Icon looks nice! Reviewing now.

**Comment by anselmbradford** [comment]
template looks like:
screen shot 2015-02-16 at 8 16 27 pm

File: CUSTOMIZE.md [hunk 1]
```diff
@@ -54,8 +54,17 @@ To test the pagination feature, you might find it handy to force a particular
...
+The root `graphics` folder contains source template files for images in the application. Using these templates you can edit and re-generate the major graphics of the application. These are provided in Adobe Illustrator and SVG formats.
+
+- **Application logo**.
+The `app_logo` file can be used to adjust the application logo in the upper-left of the site. Make changes and export and overwrite the file in `/app/assets/images/app_logo.png`.
+
+- **Map marker graphics**.
+The `map_icons_template` file can be used to adjust the map marker appearance by making changes and exporting and overwriting the files in `/app/assets/images/markers`. The Google map appears on the search results and location details pages.
+
+- **Touch icons**.
+The `touch_icons_template` file can be used to change the touch icon that show up in mobile and tablet devices when the site is saved to the home screen. Adjust the appearance by making changes and exporting and overwriting the files in `/app/assets/images/touch_icons`. The Google map appears on the search results and location details pages.
```
Comment (monfresh) [inline CUSTOMIZE.md]: I think this last sentence about Google maps doesn't belong here, right?

Comment (monfresh) [inline CUSTOMIZE.md]: Also, in the first sentence, it should be `shows up`.

Comment (anselmbradford) [inline CUSTOMIZE.md]: Cool! Pushed an edit.

**Comment by monfresh** [comment]
I tested it on my iPhone 6 and it looks great! Thanks! :shipit:

**Comment by anselmbradford** [comment]
Ready for squash and merge?

**Comment by monfresh** [comment]
Yep. :shipit: is the `ship it` icon :smile:

**Comment by anselmbradford** [comment]
haha okay

**Pull request merged**